crates/viewer/re_global_context/src/app_options.rs

```diff
@@ -67,8 +67,7 @@ pub struct AppOptions {
 impl Default for AppOptions {
     fn default() -> Self {
         Self {
-            low_latency: 0.100,
-            warn_latency: 0.200,
+            warn_e2e_latency: 1.0,
```

Comment:
I also renamed it so that the old value in peoples configs wouldn't be transferred to the new option